src/@types/secure-random/index.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module "secure-random" {
+  export function randomBuffer(length: number): Buffer;
```

Comment:
I recently found out that bcrypto has randomBytes method and it's more secure than this